src/common/imageio.c

```diff
@@ -531,6 +541,11 @@ int dt_imageio_is_hdr(const char *filename)
 #endif
 #ifdef HAVE_LIBAVIF
        || !strcasecmp(c, ".avif")
+#endif
+#ifdef HAVE_LIBHEIF
```

Comment:
We need a runtime detection if libheif has been compiled with heic support here!